_guides/getting_started.md

```diff
@@ -10,31 +10,60 @@ body_class: tutorial
 resource_icon: /svg/resources/guide.svg
 ---
 
-# A Step by Step Introduction to Linear Regression
+# Getting Started with Linear Regression and Vowpal Wabbit
 
-## A first data-set
+This guide describes how to run Linear Regression with Vowpal Wabbit (VW). It features an overview of a simple regression problem using a VW workflow tutorial with examples. Find information about interacting with VW, including structuring input, understanding output, and learn important VW diagnostics. 
```

Comment:
@kumpera comments: 
Linear Regression is a form of regression, not an algorithm per-se.
Creation of models is one of the many tasks the reader can do, it's not an end goal on itself.
I think "in VW" sounds worse than "with VW" as the result of such regression outlives the usage VW itself.

See updated. Will apply to CB guide as well.